README.md

```diff
@@ -1,73 +1,189 @@
+# [`oak` module](https://app.viam.com/module/viam/oak)
 
-This is a [Viam module](https://docs.viam.com/manage/configuration/#modules) for the [OAK](https://shop.luxonis.com/collections/oak-cameras-1) family of cameras. Registered at https://app.viam.com/module/viam/oak.
+This is a [Viam module](https://docs.viam.com/manage/configuration/#modules) that implements the [`rdk:component:camera` API](/components/camera/#api) in three models for the [OAK](https://shop.luxonis.com/collections/oak-cameras-1) family of cameras. 
+The module supports:
+- getting color frames, stereo depth frames, and point cloud data from OAK cameras using the `oak-d` and `oak-ffc-3p` component models
+- setting up a YOLO detection network on the camera VPU using the `yolo-detection-network` vision service model.
 
-## Build and Run
+> [!NOTE]  
+> Learn about the difference between modules, models, components, and services in [our docs](https://docs.viam.com/appendix/glossary/).
+
+## Requirements
+The module executable is currently only supported on `linux/arm64` and `darwin/arm64`. Make sure your machine is running on one of these architectures to avoid exec format issues.
+
+## Supported Models
 
-### Configure your camera
+### Components
+* `viam:luxonis:oak-d` - Configures the [OAK-D](https://shop.luxonis.com/products/oak-d) integrated camera model.
+* `viam:luxonis:oak-ffc-3p` - Configures the [OAK-FFC-3P](https://shop.luxonis.com/products/oak-ffc-3p) non-integrated model with flexible board and camera sensor support.
 
+Other OAK family cameras may work with the either component model, but are not officially supported and tested.
+
+### Services
+* `viam:luxonis:yolo-detection-network` - Configures [a DepthAI Yolo Detection Network](https://docs.luxonis.com/software/depthai-components/nodes/yolo_detection_network/) on the OAK device pipeline.
+
+## How to use on your own machine
 > [!NOTE]  
-> Before configuring your camera, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+> Before configuring your resource, you must create a new smart machine. See [here](https://docs.viam.com/use-cases/configure/#configure-a-machine) for how to do that and more on the setup and configuration process as a whole.
+
+Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com/).
 
-Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/).
-Click on the **Components** subtab and click **Create component**.
-Select the `camera` type, then select the `oak` model.
+[Add `camera` / `oak`,  `camera` / `oak-d` or `camera` / `oak-ffc-3p` to your machine](https://docs.viam.com/configure/#components).
 Enter a name for your camera and click **Create**.
 
-On the new component panel, copy and paste the following attribute template into your camera’s **Attributes** box:
+To configure the vision service, click on the **Services** subtab and click **Create service**.
```

Comment:
```suggestion
To configure the vision service, add [``](/configure/#services).
```
This isn't the right instructions. What would be the name of the service? I can't figure out how to add it -- doesn't seem like it's a part of this module?

Reply:
Unfortunately the new version that actually does all these changes is not released yet.

Reply:
Try it again! I just released a new version

Reply:
Also the suggested change `To configure the vision service, add [``](/configure/#services).` results in this 
<img width="341" alt="Screenshot 2024-08-13 at 4 30 19 PM" src="https://github.com/user-attachments/assets/9a9abca0-6c6d-4ac2-a5e1-467efb2a9512">